server/config/config.go

```diff
@@ -107,6 +110,15 @@ type GRPCConfig struct {
 	Address string `mapstructure:"address"`
 }
 
+// GRPCWeb defines configuration for the gRPC-web server.
+type GRPCWeb struct {
+	// Enable defines if the gRPC-web should be enabled.
+	Enable bool `mapstructure:"enable"`
+
+	// Address defines the gRPC-web server to listen on
+	Address string `mapstructure:"address"`
```

Comment:
We could also just keep the `enable` config, and if true, bind the server to the HTTP port. I'm not sure it's useful to expose so many ports. wdyt?

Reply:
I tried to bind to the HTTP server. Getting this [error](https://pastebin.com/iFNTbgcM).

Reply:
I created this issue to discuss it: https://github.com/cosmos/cosmos-sdk/issues/8254